packages/base/ethereum-address.gts

```diff
@@ -0,0 +1,60 @@
+import { isAddress, getAddress } from 'ethers';
+import { primitive, Component, CardBase, useIndexBasedKey } from './card-api';
+import { BoxelInput } from '@cardstack/boxel-ui';
+import { fn } from '@ember/helper';
+import { not } from '@cardstack/boxel-ui/helpers/truth-helpers';
+
+function isEthAddress(address: string): boolean {
+  try {
+    return isAddress(address);
+  } catch {
+    return false;
+  }
+}
+
+function isChecksumAddress(address: string): boolean {
+  try {
+    return getAddress(address) == address;
```

Comment:
Is there any particular reason we use loose equals (`==`) instead of strict equals (`===`)?

Reply:
In this case, no. I will change to strict equals

Reply:
Resolved https://github.com/cardstack/boxel/pull/442/commits/eae6629d5c22181a5281078b712cdf0012f1eac6